docs/Learning-Environment-Design-Agents.md

```diff
@@ -560,6 +562,35 @@ between -1 and 1.
  of an entity to the `BufferSensor`.
  - Normalize the entities observations before feeding them into the `BufferSensor`.
 
+### Goal Observations
+
+It is possible for agents to collect observations that will be treated as "goal".
+A goal is used to condition the policy of the Agent, meaning that if the goal
+changes, the behavior of the Agent will change as well. Note that this is true
```

Comment:
I think we should say that goals change the policy, which is a mapping from (observation -> action)